rfc/apis.md

```diff
@@ -0,0 +1,122 @@
+# Constellation APIs (v1)
+
+## Base
+
+The API base starts with `constellation` followed by the API version.
+At this moment, the only valid API version is `v1`:
+
+```
+/constellation/v1/
+```
+
+## API groups
+
+The API version is followed by the API group. Possible values are:
+
+- [`versions`: version information for Constellation components](version-api.md)
+- [`image`: metadata for individual Constellation OS images](image-api.md)
+  - `info`: image reference lookup for each cloud provider and additional metadata
+  - `measurements`: TPM measurements for Constellation OS images
+  - `raw`: raw OS images
+  - `sbom`: SBOMs for Constellation OS images
+
+There may be more API groups in the future (e.g. `cli`)
+
+## API paths overview
+
+- [`/constellation/v1/ref/<ref>/stream/<stream>/versions/latest/<kind>.json`](version-api.md#latest)
+- [`/constellation/v1/ref/<ref>/stream/<stream>/versions/major/<base>/<kind>.json`](version-api.md#major-to-minor-version-list)
+- [`/constellation/v1/ref/<ref>/stream/<stream>/versions/minor/<base>/<kind>.json`](version-api.md#minor-to-patch-version-list)
+- [`/constellation/v1/ref/<ref>/stream/<stream>/image/<version>/info.json`](image-api.md#image-lookup-table)
```

Comment:
```suggestion
- [`/constellation/v1/ref/<ref>/stream/<stream>/<version>/image/info.json`](image-api.md#image-lookup-table)
```
Might be better the other way around, especially regarding the meaning of a short path.

Reply:
This would also make deleting a single version much simpler.

Reply:
As discussed with @malt3 , I will implement this change before the next release.